packages/lib/server/getLuckyUser.ts

```diff
@@ -410,10 +415,18 @@ async function getCalendarBusyTimesOfInterval(
         user.userLevelSelectedCalendars,
         true,
         true
-      ).then((busyTimes) => ({
-        userId: user.id,
-        busyTimes,
-      }))
+      )
+        .then((busyTimes) => ({
+          userId: user.id,
+          busyTimes,
+        }))
+        .catch((error) => {
```

Comment:
Need to log each individual promise rejection in order to get identifiable information